src/components/Element/ArticleElement/Tip.jsx

```diff
@@ -1,7 +1,13 @@
 import clsx from 'clsx';
+import { useEffect, useState } from 'react';
 
 export function Tip({ word, position, link }) {
   const { top, left } = position;
```

Comment:
may be move this below useState, to keep things organised. Just a suggestion

Reply:
removed.